src/aura/BGE_EntryForm/BGE_EntryFormController.js

```diff
@@ -104,20 +106,26 @@
                         header: component.get('v.donationModalHeader'),
                         body: newcomponent,
                         showCloseButton: true,
-                        cssClass: 'slds-modal_large'
+                        cssClass: 'slds-modal_large',
+                        closeCallback: function() {
+                            component.set('v.modalOpen', false);
```

Comment:
Since "component.set('v.modalOpen', false);" is used three times here (also on lines 115 and 123), we could explore defining this as a named function within this function and then call it directly on these lines, like:

let setModalOpenToFalse = function() {
    component.set('v.modalOpen', false);
}

then you could reference it as the closeCallback function on line 110, and call it below using setModalOpenToFalse()